test.py

```diff
@@ -377,15 +377,15 @@ def helper(self, queue: Queue[Person]) -> None:
         for val in test_list:
             self.assertTrue(queue.dequeue().name, val)
 
-        for i in range(0, 1_000_000):
-            queue.enqueue(Person(name=f'{i}'))
+        # for i in range(0, 1_000_000):
```

Comment:
Why are we commenting out this test?

Reply:
Nevermind I see your comment. Instead of `1_000_000` maybe just set it to `10_000`?